keel-core/src/main/kotlin/com/netflix/spinnaker/keel/events/ResourceEvent.kt

```diff
@@ -238,22 +238,25 @@ data class ResourceActuationPaused(
  * Actuation on the managed resource has been vetoed.
  *
  * @property reason The reason why actuation was vetoed.
+ * @property vetoer The veto that vetoed this resource
```

Comment:
nit: If it's the veto, why call it `vetoer` and not `veto`?

Reply:
Actually, instead of storing the name of the veto, you could replace this with a boolean flag called `dueToMissingDependency` or similar, since that's actually what we want to check.